app/services/configuration_repository_service.rb

```diff
@@ -138,7 +138,12 @@ def remote_file_a_json_array?(file_path)
       logger.debug("Checking that #{repo_shortform}/#{file_path} is a json array")
 
       contents_map = client.contents(repo_shortform, path: file_path)
-      contents_json = contents_map[:encoding] == "base64" ? Base64.decode64(contents_map[:content]) : contents_map[:content]
+      contents_json =
```

Comment:
I feel like this is more confusing, is this auto-corrected?

Reply:
To break up the long-line ternary, rubocop suggested I use an `if/else` statement.

Reply:
I think in this case, it's more confusing.

Reply:
How do you feel about this so rubocop doesn't complain? Is this cool:

<img width="700" alt="screen shot 2018-04-03 at 2 04 54 am" src="https://user-images.githubusercontent.com/8942499/38232067-771783da-36e3-11e8-8411-ce50856d8e32.png">

Reply:
not a fan, but #yolo